automation/src/main/java/org/wso2/carbon/testgrid/automation/beans/TestNGTest.java

```diff
@@ -19,15 +19,39 @@
 package org.wso2.carbon.testgrid.automation.beans;
 
 import org.wso2.carbon.testgrid.automation.exceptions.TestGridExecuteException;
+import org.wso2.carbon.testgrid.automation.executers.common.TestExecuter;
+import org.wso2.carbon.testgrid.automation.executers.common.TestExecuterFactory;
 import org.wso2.carbon.testgrid.common.Deployment;
+import org.wso2.carbon.testgrid.common.constants.TestGridConstants;
```

Comment:
The package name should be org.wso2.testgrid. This is not a Carbon product/component.

Reply:
This cannot be fixed in this PR and will be corrected through Issue #42